v2/features/stats/platform/common.tsx

```diff
@@ -0,0 +1,67 @@
+import { PickleModelJson } from "picklefinance-core";
+import { ReceiptDataModel, VerboseReceipts } from "./types";
+import { RawChain } from "picklefinance-core/lib/chain/Chains";
+
+const PICKLE_API = `https://f8wgg18t1h.execute-api.us-west-1.amazonaws.com/prod`;
```

Comment:
Let's use the `api.pickle.finance` host.

Reply:
deleted file, not needed